bittensor/_subtensor/__init__.py

```diff
@@ -232,6 +236,9 @@ def determine_chain_endpoint(network: str):
         elif network == "nobunaga": 
             # Staging network.
             return bittensor.__nobunaga_entrypoints__[0]
+        elif network == "bellagene":
+            # Parachain test net
+            return bittensor.__bellagene_entrypoints__[0]
```

Comment:
Do we still use multiple entrypoints? I think we can probably depreciate that structure